playlists/excel.yaml

```diff
@@ -629,14 +629,14 @@
   group: Worksheet
   api_set:
     ExcelApi: 1.1
-- id: excel-shapes-create-and-delete
+- id: excel-shape-create-and-delete
```

Comment:
curious about reason for rename...?

Reply:
I wanted to be consistent with "range" and "worksheet".